app/components/Dropdown/Dropdown.tsx

```diff
@@ -0,0 +1,38 @@
+'use client'
+
+import React from 'react';
+
+// components
+import Button from '../Button/Button'
+
+// styles
+import './Dropdown.scss'
+
+const Dropdown = () => {
+    const [open, setOpen] = React.useState(false);
```

Comment:
This is a very minor thing but contributes with the code readability. Usually boolean variables should have a prefix such as `is`, `can` or `has`. In this case `isOpen`, but its up to you